src/Codeception/Module/REST.php

```diff
@@ -301,6 +301,19 @@ public function amBearerAuthenticated($accessToken)
     }
 
     /**
+     * Adds NTLM authentication via username/password.
+     *
+     * @param $username
+     * @param $password
+     * @part json
+     * @part xml
+     */
+    public function amNTLMAuthenticated($username, $password)
+    {
+        $this->client->setAuth($username, $password, 'ntlm');
```

Comment:
1. Please copy `isFunctional` check from `amHttpAuthenticated` method. (throw ModuleException if it is a functional module unless you can implement this kind of auth for framework modules).
2. Implement some way to check a version of guzzle.
3. Add comment that this function only works with Guzzle >=6.3.0